src/account.rs

```diff
@@ -92,6 +97,30 @@ pub(crate) struct Balance {
     unverified: Unverified,
 }
 
+#[derive(Debug, Args)]
+pub(crate) struct Transfer {
+    // Bech32Address of the account to transfer assets to.
+    to: String,
```

Comment:
Rather than taking  a `String` here and parsing it later, perhaps we can take a `Bech32Address` directly?

Reply:
I changed it for this PR, but the reason I wrote it this way is to possibly support using account indexes instead to quickly send between your own accounts. But we can quickly change that if we need something like this